debian/eos-install-app-helper.install

```diff
@@ -0,0 +1,7 @@
+usr/bin/*
+usr/share/eos-application-tools/icon-overrides/*
+usr/share/locale/*/*/*
+usr/share/icons/*/*/*
```

Comment:
It looks like there is one missing directory level for the icons, as they are in: usr/share/icons/hicolor/64x64/apps/

Reply:
Oh huh. It works as is - I compared the file lists. We can add another. The globbing must match dirs anyway.

Reply:
In that case, can we just say `usr/share/icons/*` and `usr/share/locale/*` ?

Reply:
sure!